### PR TITLE
Doc 9222

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -1,10 +1,306 @@
...
+| *Summary*: CPU utilization is much higher in 7.0.1/2 when cluster is idle
+
+| https://issues.couchbase.com/browse/MB-48030[MB-48030^]
+| *Summary*: [TLS] Strict mode, see errors in query log for 9101 port for indexer client
+
+| https://issues.couchbase.com/browse/MB-48144[MB-48144^]
+| *Summary*: Add version check to enable plasma indexes on Ephemeral buckets from 7.0.2
+
+| https://issues.couchbase.com/browse/MB-48145[MB-48145^]
+| *Summary*: [Enforce-TLS] IndexStatus returns Cannot communicate with indexer process. Information on indexes may be stale. Will retry
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc]: Should remove this one, Enforce TLS didn't exist before this release.

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -1,10 +1,306 @@
...
+| *Summary*: getIndexStatus endpoint is getting timed out
+
+| https://issues.couchbase.com/browse/MB-47878[MB-47878^]
+| *Summary*: CBSE - Increase the poll time in pollForDeletedBuckets at projector
+
+| https://issues.couchbase.com/browse/MB-48017[MB-48017^]
+| *Summary*: CPU utilization is much higher in 7.0.1/2 when cluster is idle
+
+| https://issues.couchbase.com/browse/MB-48030[MB-48030^]
+| *Summary*: [TLS] Strict mode, see errors in query log for 9101 port for indexer client
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc]: Should remove this one, TLS strict mode didn't exist before this release.

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -1,10 +1,306 @@
...
+Couchbase Server 7.0.2 was released in October 2021.
+This maintenance release contains new features and fixes to issues.
+
+[#new-features-improvements-702]
+=== New Features
+
+This section highlights the notable new features and improvements in this release.
+
+* TLS can be specified as mandatory for all internal and external cluster-communications &#8212; see xref:manage:manage-security/manage-tls.adoc[Manage On-the-Wire Security].
+
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc:20]: We also added a new feature to allow administrators to enable HSTS (HTTP Strict Transport Security)

File: modules/release-notes/pages/relnotes.adoc [hunk 4]
```diff
@@ -1,10 +1,306 @@
...
+| *Summary*: [TLS] Strict mode, see errors in query log for 9101 port for indexer client
+
+| https://issues.couchbase.com/browse/MB-48144[MB-48144^]
+| *Summary*: Add version check to enable plasma indexes on Ephemeral buckets from 7.0.2
+
+| https://issues.couchbase.com/browse/MB-48145[MB-48145^]
+| *Summary*: [Enforce-TLS] IndexStatus returns Cannot communicate with indexer process. Information on indexes may be stale. Will retry
+
+| https://issues.couchbase.com/browse/MB-48146[MB-48146^]
+| *Summary*: [IPV6] no index available when switching to ipv6 or ipv6only IP family
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc]: Remove this one, ipv6only didn't exist before this release.

File: modules/release-notes/pages/relnotes.adoc [hunk 5]
```diff
@@ -1,10 +1,306 @@
...
+| *Summary*: XDCR - make health_check_interval configurable
+
+| https://issues.couchbase.com/browse/MB-47246[MB-47246^]
+| *Summary*: Switch to new javascript evaluator
+
+| https://issues.couchbase.com/browse/MB-47521[MB-47521^]
+| *Summary*: XDCR - collect remote clusters and replication info as part of cbcollect
+
+| https://issues.couchbase.com/browse/MB-47546[MB-47546^]
+| *Summary*: Bind only to IPv4 addresses when invoked with IPv4-Only cluster-wide setting - xdcr
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc]: Remove this one, IPV4-Only didn't exist before this release.

File: modules/release-notes/pages/relnotes.adoc [hunk 6]
```diff
@@ -1,10 +1,306 @@
...
+
+| https://issues.couchbase.com/browse/MB-47723[MB-47723^]
+| *Summary*: Enforce TLS - XDCR
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc]: Remove this one, it's a new feature, not an issue

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: Removed.

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: Removed.

File: modules/release-notes/pages/relnotes.adoc (hunk 3 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:20]: I have added "* HSTS (HTTP Strict Transport Security) can now be enabled."

File: modules/release-notes/pages/relnotes.adoc (hunk 4 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: Removed.

File: modules/release-notes/pages/relnotes.adoc (hunk 5 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: Removed.

File: modules/release-notes/pages/relnotes.adoc (hunk 6 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: Removed.

**Comment by jon-strabala** [comment]
Hi Tony there is also an update for Eventing concerning some new functionality.

The Eventing service now supports a bucket backed caching capability to improve performance for repetitive Data Service (KV) lookups.

**Comment by tonyjhillman** [comment]
I have added:
* The Eventing Service now supports a bucket-backed caching capability, to improve performance for repetitive Data Service lookups.

**Comment by mihirkamdarqe** [comment]
**Following is the feedback for the Index service section**

https://issues.couchbase.com/browse/MB-47833 - this is a regression introduced in 7.0.2 itself, so I dont think its necessary to release note it. 
https://issues.couchbase.com/browse/MB-48144 - this is a regression introduced in 7.0.2 itself, so I dont think its necessary to release note it. 
https://issues.couchbase.com/browse/MB-48366 - this is a regression introduced in 7.0.2 itself, so I dont think its necessary to release note it. 
https://issues.couchbase.com/browse/MB-47737 - there is no functional impact. I dont see the need to release note this
https://issues.couchbase.com/browse/MB-48336 - Summary can be reworded to "Rollback stats will now be persisted in case of indexer crash" or better
https://issues.couchbase.com/browse/MB-48017 -  - this is a regression introduced in 7.0.2 itself, so I dont think its necessary to release note it. 
https://issues.couchbase.com/browse/MB-47878 - Reword it - "Increase polling interval in pollForDeletedBuckets in projector" or better
https://issues.couchbase.com/browse/MB-47760 - Reword it

**Comment by tonyjhillman** [comment]
For the Index Service, I have removed MBs 47833, 48144, 48366, 47737, and 48017, as specified. 

For 48366 and 47878, I have added the specified rewording.

For 47760, I have reworded as follows: Panic caused by extraneous unlocking.
Let me know if an alternative rewording for 47760 is required.

**Comment by arod1987** [comment]
Looks okay for XDCR and Tools

**Comment by jon-strabala** [comment]
Looks okay for Eventing

**Review by RayOffiah — APPROVED** [review]
As far as I can tell, looks okay.

**Comment by mihirkamdarqe** [comment]
Should we document https://issues.couchbase.com/browse/MB-48783 ?

**Pull request merged**